### ISSUE TITLE
Diagonal artifacts on a Radeon R9 Fury with similar modern system stats

### ISSUE THREAD

**Issue by Cervator** [MovingBlocks/Terasology#2397]
Originally posted in the forum with way more detail there:

http://forum.terasology.org/threads/strange-constant-diagonal-artifacting.1565/

Youtube video showing the issue:

https://www.youtube.com/watch?v=W9qMbXBdS0s
##

**Comment by emanuele3d** [comment]
Some interesting details:
- artifact seems to occur only in the scene reflected pass (aka water reflections) and the ambient occlusion passes (x2)
- changing the resolution factor affects the problem in a way that seems to hint to one of the buffer not being resized appropriately when the resolution changes OR, perhaps more likely, not re-fecthed from the FrameBuffersManager when the resolution changes.

**Comment by emanuele3d** [comment]
@Cervator: can you assign me and @tdgunes on it?

**Comment by emanuele3d** [comment]
Thank you @SkySom!

**Comment by TimVerhaegen** [comment]
Another case. I am not able to open my inventory either, might be connected. I was trying to place water down to replace this issue.
![screenshot_33](https://user-images.githubusercontent.com/17968114/29236162-37a505c2-7f07-11e7-8472-86d5ad939dde.png)

My PC specs:
- Windows 10 Pro 64-bit (not yet activated)
- CPU: AMD Ryzen 5 1600
- RAM: 16GB
- ASUS PRIME B360-PLUS
- GPU1: Radeon RX 480
- GPU2: AMD Radeon HD 7700 Series.
- SSD 250GB
- HDD 1TB

Modules enabled: None!

Extra info: I just launched Terasology and I got this, it could be an issue with the RX GPUs.
